README.md

```diff
@@ -37,8 +59,44 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
-## Learn More
+## Deployment
+
+Follow this Heroku link for the deployment of this Project 
+
+## Built With
+
+* [Heroku](https://devcenter.heroku.com/articles/github-integration) - The web framework used
+* [NodeJs](https://nodejs.org/en/)
+* [MongoDB](https://github.com/mongodb/docs)
+* [Express](https://expressjs.com/)
+* [React](https://github.com/reactjs/reactjs.org)
+* [Passport](https://github.com/jaredhanson/passport-github)
+* [Materialize](https://github.com/Dogfalo/materialize)
+
+
+## Contributing
+
+If you'd like to contribute, please feel free to reach out to any of the original authors!
+
+## Versioning
+
+We use [GitHub](http://github.com/) for versioning. For the versions available, see the [tags on this repository](https://github.com/center-aisle/middle-ground). 
+
+## Authors
+
+* [Naomi Woodruff](https://github.com/naywood)
+    SDE: Designed and engineered the frontend by making the pages, creating user examples, and aesthetic framework for this app using materialize.
+* [Bryan Kelley](https://github.com/bryanrkelley)
+    SDE: Designed and gngineered the frontend by making the pages, setting up the components, creating the aethetic framework for this app using materialize, and setting up the continuous integration with Heroku.
```

Comment:
mispelled engineered

Reply:
and aesthetic

Reply:
Oh, and keep all your tenses consistent - designed, engineered, set up, created, set up

Reply:
that last comment goes for the entire readme